adal/build.gradle

```diff
@@ -182,8 +182,8 @@ dependencies {
 
 task sourcesJar(type: Jar) {
     from android.sourceSets.main.java.srcDirs
-    classifier = 'sources'
-    destinationDir = reporting.file("$project.buildDir/outputs/jar/")
+    archiveClassifier  = 'sources'
+    getDestinationDirectory().set(reporting.file("$project.buildDir/outputs/jar/"))
```

Comment:
Is this related to lint work?

Reply:
code QL, it was deprecated so I change the syntax